scripts/compute_l2fc.R

```diff
@@ -1,15 +1,14 @@
+options(cli.unicode = FALSE)
+library(argparse)
+library(magrittr)
+library(tidyverse)
 suppressPackageStartupMessages(library(argparse))
-#suppressMessages(library(cmapR))
-suppressPackageStartupMessages(library(dplyr)) #n()
-#suppressPackageStartupMessages(library(scam))
-#suppressPackageStartupMessages(library(magrittr))
-#suppressPackageStartupMessages(library(tidyverse))
-#suppressPackageStartupMessages(library(reshape2))
-library(prismSeqR)
+suppressPackageStartupMessages(library(dplyr))
+source("/workspace/scripts/src/compute_l2fc.R")
```

Comment:
Should this be of the format `source("./src/compute_l2fc.R")` ?

Reply:
I prefer to be explicit in these instances so that things don't depend on the current working dir